apps/test/unit/lib/util/audioApiTest.js

```diff
@@ -79,23 +80,36 @@ describe('Audio API', function() {
   });
 
   describe('playSpeech', function() {
-    it('has two arguments, "text" and "gender"', function() {
+    beforeEach(() => {
+      replaceOnWindow('appOptions', {
```

Comment:
can we remove this now that we've fixed the binding in `audioApiDropletConfig.js`? if we do still need to mock `appOptions`, we can mock it as an empty object (i.e., `replaceOnWindow('appOptions', {})`) since we aren't using the `level` that's currently mocked on the next line